dev/com.ibm.ws.microprofile.config.1.1_fat/fat/src/com/ibm/ws/microprofile/config/fat/tests/StressTest.java

```diff
@@ -48,7 +47,7 @@
     public static LibertyServer server;
 
     @ClassRule
-    public static RepeatTests r = RepeatConfigActions.repeat("StressServer", Version.CONFIG13_EE7, Version.LATEST);
+    public static RepeatTests r = MicroProfileActions.repeat("StressServer", MicroProfileActions.MP20, MicroProfileActions.LATEST);
```

Comment:
The old config was equivalent to MP14, rather than MP20. Probably unimportant but was this a deliberate change?

Reply:
This one is a mistake, will change it

Reply:
done